### PR TITLE
Bug/gh66 Verify that iterationSetup.isDeleted behavior is as per 10 25; fixes #66

### PR DESCRIPTION
**Pull request by kamilatrhea**
Integration test for validation of iterationSetup isDeleted property behavior added.

### CODE REVIEW

File: IntegrationTestSuite/WebservicesIntegrationTests/packages.config [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
+  <package id="DotNetZip" version="1.13.7" targetFramework="net452" />
   <package id="Microsoft.AspNet.WebApi.Client" version="5.2.7" targetFramework="net452" />
-  <package id="Newtonsoft.Json" version="11.0.2" targetFramework="net452" />
-  <package id="NLog" version="4.5.2" targetFramework="net452" />
-  <package id="NUnit" version="3.10.1" targetFramework="net452" />
-  <package id="NUnit3TestAdapter" version="3.10.0" targetFramework="net452" />
+  <package id="Newtonsoft.Json" version="12.0.3" targetFramework="net452" />
+  <package id="NLog" version="4.7.0" targetFramework="net452" />
+  <package id="NUnit" version="3.12.0" targetFramework="net452" />
+  <package id="NUnit3TestAdapter" version="3.16.1" targetFramework="net452" developmentDependency="true" />
```
Comment (lxatrhea) [inline IntegrationTestSuite/WebservicesIntegrationTests/packages.config:8]: Was this change intentional?
What does it mean?

Comment (kamilatrhea) [reply IntegrationTestSuite/WebservicesIntegrationTests/packages.config:8]: This is, because I updated nuget packages in my solution.

Comment (lxatrhea) [reply IntegrationTestSuite/WebservicesIntegrationTests/packages.config:8]: I saw that, but why is the developmentDependency="true" added?

Comment (kamilatrhea) [reply IntegrationTestSuite/WebservicesIntegrationTests/packages.config]: changed
